docs/content/dagster-cloud/insights/integrating-snowflake-and-dbt.mdx

```diff
@@ -29,23 +29,47 @@ To complete the steps in this guide, you'll need:
 
 ---
 
-## Step 1: Instrument your dbt asset definition
+## Step 1: Instrument your Dagster code
 
+<TabGroup>
+<TabItem name="Using assets">
```

Comment:
```suggestion
<TabItem name="Using assets">

```

`TabItem`s need a blank space after them in order for the Markdown to render correctly.